ext/nnstreamer/tensor_filter/tensor_filter_llamacpp.cc

```diff
@@ -14,8 +14,8 @@
  * @details Pipeline example
  *          gst-launch-1.0 filesrc location=input.txt ! application/octet-stream !
  *          tensor_converter ! other/tensors,format=flexible !
- *          tensor_filter framework=llamacpp model=llama-2-7b-chat.Q2_K.gguf invoke-dynamic=TRUE !
- *          other/tensors,format=flexible ! tensor_decoder mode=octet_stream !
+ *          tensor_filter framework=llamacpp model=llama-2-7b-chat.Q2_K.gguf invoke-dynamic=TRUE
+ *          custom=num_predict=32 ! other/tensors,format=flexible ! tensor_decoder mode=octet_stream !
```

Comment:
How about use `key:value` syntax for custom props like `custom=num_predict:32,k1:v1,...` same for other subplugins?

Reply:
Since I was referenced to mxnet, I didn't know that most of the subplugins use `key:value` format 😅
Updated it
Thank you :)

Reply:
Oh we might standardize all the subplugins someday..